recipes/usalign/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set name = "USalign" %}
+{% set version = "20240303" %}
+{% set sha256 = "b4da9fc58449f96a98eabf4137082cfc161787a5df739df351186e70b9c87950" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://zhanggroup.org/US-align/bin/module/USalignLinux64.zip
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - sysroot_linux-64  # [linux]
+
+test:
+  commands:
+    - which USalign
```

```suggestion
    - USalign --help
```

```suggestion
    - USalign --version
```
Do something like this instead because the binaries is actually executed. If you want to check that it's the correct USalign, you should use an existence check to `$PREFIX/bin/USalign`.